pygmt/tests/test_plot.py

```diff
@@ -95,7 +95,8 @@ def test_plot_fail_no_data(data):
 
 def test_plot_fail_size_color(data):
     """
-    Should raise an exception if array sizes and color are used with matrix.
+    Should raise an exception if array color, sizes and intensity are used with
+    matrix.
     """
     fig = Figure()
```

Comment:
The three tests can be simplified using (I didn't check the codes carefully, please double-check it that it's correct):
```
kwargs = dict(data=data, region=region, projection="X10c", frame="afg")
with pytest.raises(GMTInvalidInput):
    fig.plot(style="c0.2c", color=data[:, 2], **kwargs)
with pytest.raises(GMTInvalidInput):
    fig.plot(style="cc", sizes=data[:, 2], color="red", **kwargs)
with pytest.raises(GMTInvalidInput):
    fig.plot(style="c0.2c", color="red", intensity=data[:, 2], **kwargs)
```

Reply:
done in d6ec990.